tests/test-vip-plugins.php

```diff
@@ -108,6 +108,11 @@ public function test__modify_active_plugins() {
 		 */
 		$option_db = $wpdb->get_var( "SELECT option_value FROM $wpdb->options WHERE option_name = 'active_plugins' LIMIT 1" );
 		$option_db = maybe_unserialize( $option_db );
+		$plugin_change = array_merge( $this->option_active_plugins, array( 'amp-wp/amp.php' ) );
+		// because this was a dupe plugin we will be smart enough to remove it from the option
+		if ( ( $key = array_search( 'msm-sitemap/msm-sitemap.php', $plugin_change, true ) ) !== false ) {
```

Comment:
This dynamic approach to generating control group of plugins might be needed now as we have this one huge test data situation here. But we might want to hardcore the values during the unit tests refactor, in order to have better overview of what is happening.